orchagent/vxlanorch.h

```diff
@@ -94,14 +105,44 @@ class VxlanTunnel
         return ids_.tunnel_id;
     }
 
-    sai_object_id_t getDecapMapId() const
+    sai_object_id_t getDecapMapId(tunnel_map_type_t mapType) const
     {
-        return ids_.tunnel_decap_id;
+        if (mapType == TUNNEL_MAP_T_VLAN)
```

Comment:
Avoid using ```if else```, according to the another comment):
https://github.com/volodymyrsamotiy/sonic-swss/pull/2/files#r624532442
> Use ```map``` to store mappers IDs in order to optimize the way how it is stored and also to avoid many ```if``` statements in a code that uses these variables.